src/tests/no-flowmax-in-forwardref.coffee

```diff
@@ -0,0 +1,70 @@
+{rules: {'no-flowmax-in-forwardref': rule}} = require '..'
+{RuleTester} = require 'eslint'
+
+ruleTester = new RuleTester()
+
+error =
+  message: "flowMax() cannot be used within forwardRef()"
```

Comment:
Tests are failing since this error message needs to be updated (to match the updated error message in the rule)